nohup: ignoring input
Segmentation fault (core dumped)
computing class 0...
computing class 1...
computing class 2...
computing class 3...
computing class 4...
computing class 5...
computing class 6...
computing class 7...
computing class 8...
computing class 9...
Traceback (most recent call last):
  File "pocket_match.py", line 80, in <module>
    scores.append(pocket_match(in_pocket, in_pocket_cf, label_pocket, label_pocket_cf))
  File "pocket_match.py", line 16, in pocket_match
    p = subprocess.run('./glosa -s1 {} -s1cf {} -s2 {} -s2cf {} -o 0'.format(pocket, pocket_cf, label_pocket, label_pocket_cf),
  File "/home/wentao/miniconda3/envs/pytorch-1/lib/python3.8/subprocess.py", line 512, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command './glosa -s1 ./all_pockets/3pdbA00.pdb -s1cf ./all_pockets/3pdbA00-cf.pdb -s2 ./label_pockets/2ddoA00.pdb -s2cf ./label_pockets/2ddoA00-cf.pdb -o 0' returned non-zero exit status 139.
